core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -630,128 +673,115 @@ void freeze() {
     }
 
     /***
-     * Copies items from srcChunk performing compaction on the fly.
+     * Copies entries from srcChunk performing only entries sorting on the fly
+     * (delete entries that are removed as well).
      * @param srcChunk -- chunk to copy from
-     * @param ei -- start position for copying
-     * @param maxCapacity -- max number of items "this" chunk can contain after copy
-     * @param maxBytes -- max number of key bytes "this" chunk's key manager can contain after copy
+     * @param srcEntryIdx -- start position for copying
+     * @param maxCapacity -- max number of entries "this" chunk can contain after copy
      * @return key index of next to the last copied item, NONE if all items were copied
      */
-    final int copyPart(Chunk srcChunk, int ei, int maxCapacity, int maxBytes) {
+    final int copyPartNoKeys(Chunk srcChunk, int srcEntryIdx, int maxCapacity) {
```

Comment:
I expected this method to be the same excluding the lines that did the copying of the keys 
but it is more involved
not sure I followed all changes

Reply:
Since we see copying has positive affect for writes and negative affect for scans -- is it possible to have copy/no copy controlled by some configuration knob. Maybe hard coded for simplicity. It would be good to see how this change affects cluster level benchmarks.

Reply:
1) Previously this method was also sorting the keys while copying so that is why it is not just removing some lines. I spent long time on this method and on testing it, hope it is correct.

2) As far as it is not clear what exactly is causing scan performance degradation I do not think it is reasonable to make this complex change.